__stories__/content.component.tsx

```diff
@@ -0,0 +1,206 @@
+import { storiesOf } from '@storybook/react'
```

Comment:
feel free to split up this file into multiple files as soon as one file gets too annoying